### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Sergej161995**
[DEMO LINK](https://Sergej161995.github.io/layout_moyo-header/)
 [TEST REPORT LINK](https://Sergej161995.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,14 +1,51 @@
...
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
```
Comment (marina-tilniak) [inline src/index.html:15]:
```suggestion
    <header class="header">
```

File: src/style.css [hunk 2]
```diff
@@ -1 +1,75 @@
...
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.header {
+  max-width: 1200px;
```
Comment (marina-tilniak) [inline src/style.css:16]: навіщо це?

**Review by marina-tilniak — CHANGES_REQUESTED** [review]
на великих екранах хедер має займати всю ширину стрінки 
image

File: src/index.html [hunk 3]
```diff
@@ -1,14 +1,51 @@
...
+
+    <header>
+      <img
+        src="./images/logo.png"
+        class="logo"
+        alt="logo"
+      >
+      <nav>
+        <ul class="nav__menu">
+          <li class="menu">
```
Comment (ivanchuk-yurii) [inline src/index.html:23]: I think it's `menu-item`, not `menu`

File: src/index.html [hunk 4]
```diff
@@ -1,14 +1,51 @@
...
-    <h1>Moyo header</h1>
+
+    <header>
+      <img
+        src="./images/logo.png"
+        class="logo"
+        alt="logo"
+      >
+      <nav>
+        <ul class="nav__menu">
```
Comment (ivanchuk-yurii) [inline src/index.html:22]: you don't have `nav` block, so you can't use `nav__menu` element here. rename this or add class `.nav` to parent element

File: src/index.html [hunk 5]
```diff
@@ -1,14 +1,51 @@
...
+          <li class="menu">
+            <a class="nav__link is-active">APPLE</a>
```
Comment (ivanchuk-yurii) [inline src/index.html:24]: you can't have `nav__link` element here because it's child of `menu` block

File: src/style.css [hunk 6]
```diff
@@ -1 +1,74 @@
...
+  margin: 0;
+}
+
+.menu {
+  margin-right: 20px;
+}
+
+.menu:last-child {
+  margin-right: 0;
+}
```
Comment (ivanchuk-yurii) [inline src/style.css:41]:
```suggestion
.menu:not(:last-child()) {
  margin-right: 20px;
}
```

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]

**Pull request closed without merging**